missilewars-plugin/src/main/java/de/butzlabben/missilewars/listener/GameListener.java

```diff
@@ -246,7 +246,7 @@ public void onRespawn(PlayerRespawnEvent e) {
             e.setRespawnLocation(t.getSpawn());
             FallProtectionConfiguration fallProtection = getGame().getArena().getFallProtection();
             if (fallProtection.isEnabled())
-                new RespawnGoldBlock(e.getPlayer(), fallProtection.getDuration(), getGame());
+                new RespawnGoldBlock(e.getPlayer(), fallProtection.getDuration(), fallProtection.isMessage_only_on_start(), getGame());
```

Comment:
This call has to be renamed, as the automatic getter generated by lombok ist now named isMessageOnlyOnStart(). With the current version, the project won't even compile

Reply:
-.- Sorry.